tests/acceptance/custom-user-test.js

```diff
@@ -0,0 +1,38 @@
+import { test, module } from 'qunit';
+import startApp from '../helpers/start-app';
+import destroyApp from '../helpers/destroy-app';
+import config from 'dummy/config/environment';
+import Bugsnag from 'bugsnag';
+
+module('Acceptance | custom user', {
+  beforeEach() {
+    config.bugsnag.releaseStage = 'production';
+    this.cachedEnvValue = config.environment; // NOTE: cached so we can restore it
+    config.environment = 'addon-test';
+    this.application = startApp();
+  },
+
+  afterEach() {
+    delete config.bugsnag.releaseStage;
+    config.environment = this.cachedEnvValue; // NOTE: restore original value
+    destroyApp(this.application);
+  }
+});
+
+test('with custom user set', function(assert) {
+  visit('/');
+  click('button');
+
+  andThen(function() {
+    const expected = {
+      id: 123,
+      name: 'Dummy User',
+      email: 'dummy@example.com',
+    };
+    assert.deepEqual(
```

Comment:
Yeah I was wonder about that. Ideally I wanted to test a) app doesn't have util defined, b) app does have util, but doesn't have method, c) app has util and method. But I'm not sure how to dynamically set utils on the app instance in tests.